rateapp/src/components/Search.vue

```diff
@@ -0,0 +1,146 @@
+<template>
+    <div class="search">
+        <h2 v-if="!this.subquery" class="text-2xl px-4 pt-4 font-bold">Zoeken</h2>
+        <div v-if="!this.subquery && this.$store.getters['rating/ratingQuery'].length > 0 && this.$store.getters['auth/isLoggedIn']"

```

Comment:
Dit zou wat mij betreft ook echt een functie moeten zijn. Liefst ook met checks of ratingQuery daadwerkelijk een instance is met een length property. Het gaat goed in dit geval.

Reply:
Ik heb deze voor nu allemaal gerefactored naar computed functions. Had nog een computed functie kunnen toevoegen waarin deze samengevoegd worden maar is nu in ieder geval al een stuk leesbaarder. Voor nu oke denk ik, maar kan zeker netter.